File: PHP/Ram.php
<?php

class Ram{

    private $size;
    private $price;
    
   

    public function __construct($size = 0, $price = 0) {
    
        $this->size = $size;
        $this->price = $price;

    }

    public function setRamSize($size) { //
        $this->size = $size;
    }

    public function getRamSize() { 
        return $this->size;
    }



    public function setRamprice($price) { //
        $this->price = $price;
    }

    public function getRamPrice(){ 
        return $this->price;

    } 

}
?>

File: java/Ram.java
public class Ram {
    int size;
    int price;


    Ram(){

    }

    Ram(int s, int p){
        this.size = s;
        this.price = p;
    }

    void setRamSize(int s){
        this.size = s;
     
    }

    void setRamPrice(int p){
        this.price = p;
    }

    int getRamSize(){
        return this.size;
    }

    int getRamPrice(){
        return this.price;
    }

}
